lib/pbench/server/api/__init__.py

```diff
@@ -48,59 +49,16 @@ def register_endpoints(api, app):
     api.add_resource(GraphQL, f"{app.config['REST_URI']}/graphql")
 
 
-def create_app():
+def create_app(server_config=ProdServerConfig):
```

Comment:
I'd rather see you pass in an _object_ rather than a _class_ that gets instantiated here. That just feels awkward; and passing an object will provide more flexibility for testing, too.

Reply:
you mean something like `create_app(server_config=ProdServerConfig()):` but then what do you mean by instantiating it? The config classes ideally wouldn't have any methods just constant variables